mabl functions/currentUrl.js

```diff
@@ -0,0 +1,7 @@
+/**
```

Comment:
nit: I think this file should be named `getCurrentUrl` to better describe the function this file contains.